packages/webapp/src/containers/filterSlice.js

```diff
@@ -1,4 +1,19 @@
-import { createSlice } from '@reduxjs/toolkit';
+/*
+ *  Copyright 2019-2022 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import { createSlice, current } from '@reduxjs/toolkit';
```

Comment:
As far as I can tell, `current` is unused. If that is the case, can you get rid of the import?